packages/hdwallet-ledger-webusb/src/adapter.ts

```diff
@@ -45,56 +51,60 @@ export class WebUSBLedgerAdapter {
   private async handleDisconnectWebUSBLedger(e: USBConnectionEvent): Promise<void> {
     if (e.device.vendorId !== VENDOR_ID) return;
 
+    this.keyring.emit(
+      [e.device.manufacturerName ?? "", e.device.productName ?? "", core.Events.DISCONNECT],
+      e.device.serialNumber
+    );
+
```

Comment:
Always emit native disconnect events to be handled if we wish to - shouldn't really be needed anymore given we open/close JIT